builder/build.sh

```diff
@@ -100,6 +121,8 @@ compile_r() {
   R_PRINTCMD=/usr/bin/lpr \
   R_UNZIPCMD=/usr/bin/unzip \
   R_ZIPCMD=/usr/bin/zip \
+  SHLIB_CFLAGS="${extra_cflags}" \
+  SHLIB_FFLAGS="${extra_fflags}" \
```

Comment:
I'm not 100% sure if `SHLIB_CFLAGS` and `SHLIB_FFLAGS` are the best environment variables to use, but they're the only way I've found to append CFLAGS and FFLAGS. Otherwise, you'll need to manually specify the CFLAGS and FFLAGS that R adds, which can differ across R versions.

Reply:
@glin I think this also fixes the build errors with Ubuntu 22.04, right? Currently the builds for R 3.6.1 and below are missing for Ubuntu 22.04 AFAICT.

Reply:
Yup, I'll be sure to rebuild the failing R <= 3.6.1 builds for Ubuntu 22 when this gets deployed.